examples/gcf-pubsub-vm-delete-event-handler/main.py

```diff
@@ -146,18 +207,22 @@ def delete_record(project: str, managed_zone: str, record: dict, http=None):
         managedZone=managed_zone,
         body=change)
     response = request.execute()
-    log.debug(json.dumps({
-        'message': 'END Deleting record',
+    structured_log_data = {
```

Comment:
Need to add event_id to the structure.  Also need to log event_id at info level unstructured in the execution_id filter view so the execution can be correlated to the event results report.